platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInstallCommand.java

```diff
@@ -127,42 +137,39 @@ protected void doExecute(
         | IllegalArgumentException e) {
       printError(RESTART_WARNING);
       throw e;
+    } finally {
+      SecurityLogger.audit(
```

Comment:
❗️ We are auditing even if the above throws an exception. In such case, we should audit the failure and not the started.

Reply:
❓ Should we be just auditing the profile that is being installed as opposed to the count?

Reply:
Auditing the profile seems like it might be cleaner. I'd be curious to see what @stustison would prefer before i change it to doing just that though 😄

Reply:
Profile is fine.

Reply:
Done